client/sources/common/importing.py

```diff
@@ -3,7 +3,9 @@
 
 def load_module(filepath):
     module_name = path_to_module_string(filepath)
-    return importlib.import_module(module_name)
+    module = importlib.import_module(module_name)
+    importlib.reload(module)
```

Comment:
`importlib.reload` is new in version Python3.4. I see that `imp.reload` is deprecated as of 3.4, though.